sdk/eventhub/azure-eventhub/azure/eventhub/_utils.py

```diff
@@ -245,12 +258,84 @@ def get_last_enqueued_event_properties(event_data):
         return event_data._last_enqueued_event_properties
     return None
 
+
 def parse_sas_credential(credential):
     # type: (AzureSasCredential) -> Tuple
     sas = credential.signature
-    parsed_sas = sas.split('&')
+    parsed_sas = sas.split("&")
     expiry = None
     for item in parsed_sas:
-        if item.startswith('se='):
+        if item.startswith("se="):
             expiry = int(item[3:])
     return (sas, expiry)
+
+
+def transform_single_message_if_needed(message, message_type):
```

Comment:
I think we should probably rename this method just to `transform_outbound_single_message`.

`if_needed` doesn't reflect the thing we're doing here